WordPress/openverse
# Convert VPill and VItemGroup stories to mdx

## Fixes

Related to #584 by @dhruvkb 

## Description

This PR updates stories of VPill and VItemGroup to use the composition API (MDX). Here is the screenshot of the converted VPill and VItemGroup to MDX in the storybook:
<img width="1617" alt="Screenshot 2023-03-30 at 16 19 48" src="https://user-images.githubusercontent.com/54465154/228872834-0f59c040-83b6-4c9e-83fc-a445f82fca10.png">
<img width="1607" alt="Screenshot 2023-03-30 at 16 33 06" src="https://user-images.githubusercontent.com/54465154/228872868-66071706-f756-402a-9f3f-2499a386977d.png">


## Testing Instructions

Run the Storybook from this PR to see that VPill and VItemGroup are working correctly in the storybook.

## Checklist



      `Update index.md`).
      a parent feature branch.
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin



<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
